apis/v1beta1/types.go

```diff
@@ -172,6 +172,13 @@ type VirtualMachineCloneSpec struct {
 	// virtual machine is cloned.
 	// +optional
 	MemoryMiB int64 `json:"memoryMiB,omitempty"`
+	// MemoryReservationLockedToMax is a flag that indicates whether or not the
```

Comment:
```suggestion
	// MemoryReservationLockedToMax indicates whether or not the
```

please update it everywhere.